lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/RetryPolicy/Exponential/ExponentialRetryPolicyConfiguration.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Formats;
+using Org.Apache.REEF.Tang.Util;
+
+namespace Org.Apache.REEF.IO.FileSystem.AzureBlob.RetryPolicy.Exponential
+{
+    public sealed class ExponentialRetryPolicyConfiguration : ConfigurationModuleBuilder
+    {
+        /// <summary>
+        /// Client can set implementation for this parameter
```

Comment:
Please elaborate to real documentation.

Reply:
fixed

Reply:
I would suggest matching the ExponentialRetryPolicParameterNames "The exponential retry count". Also, remove the "Client can set implementation for this parameter." text.

---
In reply to: [193574467](https://github.com/apache/reef/pull/1470#discussion_r193574467) [](ancestors = 193574467)

Reply:
fixed

---
In reply to: [193890781](https://github.com/apache/reef/pull/1470#discussion_r193890781) [](ancestors = 193890781,193574467)